ui/src/components/Canvas.tsx

```diff
@@ -531,18 +531,35 @@ const CodeNode = memo<Props>(function ({
     }
   }, [data.parent, setPodParent, id]);
 
+  const getPicture = async (pod) => {
+    const requesOptions = {
+      method: "POST",
+      headers: { "Content-Type": "application/json" },
+      body: JSON.stringify({ code: pod.content }),
+      mode: "no-cors" as RequestMode,
```

Comment:
CORS seems problematic. Let's add an Nginx route.

```shell
# nginx.conf
location /carbon {
        rewrite ^/carbon(.*)$ $1 break;
        proxy_pass http://carbon:3000;
}
```

and in compose.yml, add a container:

```
carbon:
    image: local/carbonara
```

The request URL will be `http://localhost/carbon/api/cook`